### DISCUSSION TITLE
Linux Mint 21.3 Cinnamon: Quit dialog no longer: Always on Top

### DISCUSSION THREAD

**Discussion by mlavi** [linuxmint/discussions#312] [Questions] [answered]
Hello, I recently upgraded from 21.2 to 21.3 and it was flawless. I'm now on Cinnamon 6.0.4.
It is amazing how great the upgrade experience was!

From the LinuxMint (Start) menu, choose the Quit (red power off icon) item. It displays the "Shut down this system now?" dialog, showing choices for: Suspend, Cancel, Restart, and defaulting to the Shut Down button in red. 

Before: the dialog would remain on top of other windows.
Now: it's a normal window, so when I mouse over other windows to reach the dialog (I have Window Focus: automatically raise window enabled), they cover it up.

I believe I've encountered a regression and I wanted to ask if others recall a different behavior when quitting Linux Mint Cinnamon?

**Comment by mtwebster** [comment, marked as answer]
Hi, it was a regression, thanks
Fixed now: https://github.com/linuxmint/cinnamon-session/commit/bab87485cb7a4d67c1ce8bbae51efc5195c9d6d2

**Reply by mlavi** [reply]
Thanks so much for the rapid response+fix!

**Reply by mlavi** [reply]
I'll confirm after I see the patch rollout, but I also noticed that currently, the dialog window in the panel uses the generic "gear" icon.
![panel-session-dialog](https://github.com/linuxmint/discussions/assets/5642075/c54118c1-f1ea-4f0a-ac9e-883bba837e4d)
